types/cannon/index.d.ts

```diff
@@ -845,6 +849,35 @@ declare module CANNON {
 
     }
 
+    export class Trimesh extends Shape {
+
+        vertices: number[]
+        indices: number[]
+        scale: Vec3
+
+        constructor(vertices: number[], indices: number[]);
+
+        updateTree(): void;
+        getTrianglesInAABB(aabb: AABB, result: []): [];
```

Comment:
Looks like `result` is a `number[]`

https://github.com/schteppe/cannon.js/blob/569730f94a1d9da47967a24fad0323ef7d5b4119/src/shapes/Trimesh.js#L132

Reply:
Would it not be safer to return `unknown[]` rather than just `[]` (an empty tuple)?

Reply:
`number[]` works

Reply:
Thanks for updating the argument type. What about the return type? At the moment `[]` says that the function returns an empty tuple (an empty array), not `unknown[]` or `any`[].

I would opt for returning `unknown[]`

Reply:
I ended up returning `number[]` instead of unknown[]
I looked at code,
https://github.com/schteppe/cannon.js/blob/569730f94a1d9da47967a24fad0323ef7d5b4119/src/shapes/Trimesh.js#L134-L151
and it calls the `aabbQuery` which returns a subset of the original result passed in which was a number[]. So I did that instead. I checked cannon-es and that returns `number[]` aswell.